platformio.ini

```diff
@@ -14,5 +14,5 @@ board = lolin_d32_pro
 framework = arduino
 
 ; If Platformio gives a upload error, please specify this for your respective COM port
-upload_port = /dev/ttyUSB0
+upload_port = COM7
```

Comment:
Change this back, is dependent on personal machnine, so not useful to push to develop.

Reply:
Yes on all prototypes board the code works.

Reply:
Great!